doc/TESTING.md

```diff
@@ -1,48 +1,72 @@
+     - [open-wc/testing](#open-wc/testing)
+  - [How to write tests](#how-to-write-tests)
+  - [How to run tests](#how-to-run-tests)
 
 ## Different kind of tests
+Component test - It ensures that each component render correctly and respond to user interactions as expected in isolation.
+
+In future other testing types like Integration Testing, Performance Testing can be added in the Continuous Integration process.
 
-This section should display all the different kinds of tests that you write in your project. Explain what each kind of test is for and why you use them in your project. You can also give an outlook on new tests that should be added in the future of your project. If you have a QA in place, also mention what kind of tests they perform and add contact persons.
+## Testing strategy
+The tests are crafted to validate the functionality of the software, ensuring it operates as designed and identifying bugs and errors before they are deployed to production. Testing serves as a proactive measure to identify issues early in the development phase. Our objective is to create component test for each individual component within the codebase, guaranteeing their intended functionality and preventing regressions from new changes.
 
 ## Tooling
+### open-wc/testing
+This repository contains tests written using the `@open-wc/testing` library. `@open-wc/testing` provides a suite of
+tools and utilities for testing web components, following modern best practices and standards.
 
-We're using [web-test-runner](https://modern-web.dev/docs/test-runner/overview/) to run our tests. It enables us to use a brought variety of testing tools provided by [open-wc/testing](https://open-wc.org/docs/testing/testing-package/).
+[Read more here](https://open-wc.org/docs/testing/testing-package/).
 
 ## How to write tests
-
-### ... within the open-wc testing framework
-
-All tests need to be located within the component folder inside a `test` folder. Otherwise the test-runner won't be able to locate the test. [open-wc/testing](https://open-wc.org/docs/testing/testing-package/) offers various tools for testing, such as snapshot testing or accessibility testing of the web-component.
-
-### ... with pa11y-ci for accesibility testing
-
-For standard testing, you just have to run `npm run test:a11y`.
-
-Additionaly, if you want to write custom test cases, the file `src/components/pa11y-custom.config.js` exists. It allows you to add an array of additional `parameters` to the story you're going to test.
-
-```js
-{ story: 'BlrTextButton--blr-text-button', parameters: ['icon:boilerChevronDownGreen'] }
+To write tests using `@open-wc/testing`, follow these steps:
+
+1. Import the necessary functions and utilities from `@open-wc/testing`.
+2. Write your test cases using the provided utilities such as `fixture`, `html`, `litFixture`, etc.
+
+## Directory Structure
+- `src/components/`: Contains the source code of the components and respective test for it.
+
+## Examples
```

Comment:
If "Directory Structure" and "Examples" these are sub-section of "How to write tests", you should make them a h3.

Also please add them to the Table of Content.

Reply:
Adjusted now.